### PR TITLE
add homework6 18.12.23

### PR DESCRIPTION
**Pull request by olemykha**
all tasks have been implemented.

_also I need to update isort. because this module was erroneously throwing errors.
I used for this `isort -rc your_project_directory`._

### CODE REVIEW

File: homework/homework181223.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    ('Ivan', 29, 160, 'Brown', 'male'),
+    ('Julie', 55, 90, 'Clark', 'female'),
+    ('Kenny', 38, 140, 'Harris', 'male'),
+    ('Luna', 55, 170, 'Smith', 'female'),
+    ('Mike', 55, 55, 'Johnson', 'male'),
+]
+
+# Task 1. Sorting the list by age and sex fields.
+sorted_people_list = sorted(init_people_list, key=itemgetter(1, 4))
+logging.info('\n' + '\n'.join(map(str, sorted_people_list)))
```
Comment (antohaUa) [inline homework/homework181223.py]: f-string please

File: homework/homework181223.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+# Task 2. A new list without the first two elements and the last two elements
+people_list_cut = init_people_list[2:-2]
+logging.info('\n' + '\n'.join(map(str, people_list_cut)))
+
+# Task 3. Calculating total numbers of "female" and "male".
+male_count = sum(1 for *_, gender in init_people_list if gender == 'male')
```
Comment (antohaUa) [inline homework/homework181223.py]: Pay attention that result table should be calculated on list after cut

**Review by antohaUa — CHANGES_REQUESTED** [review]
small fixes required

File: homework/homework181223.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+# Task 1. Sorting the list by age and sex fields.
+sorted_people_list = sorted(init_people_list, key=itemgetter(1, 4))
+logging.info(f'\n{pformat(sorted_people_list)}')
+
+# Task 2. A new list without the first two elements and the last two elements
+people_list_cut = init_people_list[2:-2]
+logging.info(f'\n{pformat(people_list_cut)}')
+
+# Task 3. Calculating total numbers of "female" and "male".
+male_count = [
```
Comment (antohaUa) [inline homework/homework181223.py:82]: No sense in additional list construction 
`male_count = sum(1 for *_, gender in init_people_list if gender == 'male')`

File: homework/homework181223.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    sum(1 for *_, gender in init_people_list if gender == 'male'),
+]
+female_count = [
```
Comment (antohaUa) [inline homework/homework181223.py:85]: You still count records vs init list but according task description we need to count records vs  modified list

**Review by antohaUa — APPROVED** [review]
Good

File: homework/homework181223.py (hunk 4 above)
Comment (olemykha) [reply homework/homework181223.py:85]: I will change it

**Review by antohaUa — APPROVED** [review]
Good

**Pull request merged**